Sources/Private/MainThread/NodeRenderSystem/Nodes/ModifierNodes/RoundedCornersNode.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  RoundedCornersNode.swift
+//  Lottie
+//
+//  Created by Duolingo on 10/31/22.
```

Comment:
You work at Duolingo? Cool!

I have a 1083 day streak learning Spanish 😄 Really love what Duolingo has been doing with Lottie lately -- cutting edge stuff. The new feature where the characters lip-sync with the text-to-speech synth honestly blew my mind. Is that using Lottie or some sort of custom lip sync renderer?

Reply:
Yeah! Glad you're enjoying that! I implemented that on iOS actually :D. We couldn't figure out a way to do that with Lottie so we ended up having to use a different more state-machine based animation Library

Reply:
So cool, awesome work! (Not surprised that isn't possible with Lottie)

Are you on Twitter? Would love to give you a follow if so -- I'm https://twitter.com/calstephens98

Reply:
I don't use it much for work-related stuff but gave you a follow!